set-balance.md

```diff
@@ -253,21 +259,36 @@ of the transfer, the account will be reaped.
 The dispatch origin for this call must be `Signed` by the transactor.
 
 ```k
-    syntax ExitenceRequirement ::= ".AllowDeath"
-                                 | ".KeepAlive"
+    syntax ExistenceRequirement ::= ".AllowDeath"
+                                  | ".KeepAlive"
```

Comment:
I would just name these `AllowDeath` and `KeepAlive`. Usually the `.NAME` convention is used for optionally empty collections.